README.md

```diff
@@ -127,6 +129,112 @@ val solr = AsyncSolrClient.Builder("http://localhost:8983/solr/collection1")
             .build
 ```
 
+### Adding Data to Solr
+
+[java]
+```java
+import static java.util.Arrays.asList;
+import io.ino.solrs.JavaAsyncSolrClient;
+import org.apache.solr.common.SolrInputDocument;
+
+JavaAsyncSolrClient solr = JavaAsyncSolrClient.create("http://localhost:8983/solr/collection1");
+SolrInputDocument doc1 = new SolrInputDocument();
+doc1.addField("id", "id1");
+doc1.addField("name", "doc1");
+doc1.addField("cat", "cat1");
+doc1.addField("price", 10);
+SolrInputDocument doc2 = new SolrInputDocument();
+doc2.addField("id", "id2");
+doc2.addField("name", "doc2");
+doc2.addField("cat", "cat1");
+doc2.addField("price", 20);
+solr.addDocs(asList(doc1, doc2))
+    .thenCompose(r -> solr.commit())
+    .thenAccept(r -> System.out.println("docs added"));
+```
+
+[scala]
+```scala
+import io.ino.solrs.AsyncSolrClient
+import io.ino.solrs.future.ScalaFutureFactory.Implicit // or TwitterFutureFactory
+import org.apache.solr.common.SolrInputDocument
+
+val solr = AsyncSolrClient("http://localhost:8983/solr/collection1")
+val doc1 = new SolrInputDocument()
+doc1.addField("id", "id1")
+doc1.addField("name", "doc1")
+doc1.addField("cat", "cat1")
+doc1.addField("price", 10)
+val doc2 = new SolrInputDocument()
+doc2.addField("id", "id2")
+doc2.addField("name", "doc2")
+doc2.addField("cat", "cat1")
+doc2.addField("price", 20)
+for {
+  _ <- solr.addDocs(docs = Iterable(doc1, doc2))
+  _ <- solr.commit()
+} yield print("docs added")
+```
+
+#### Directly adding POJOs to Solr
+
+[java]
+```java
+import static java.util.Arrays.asList;
+import io.ino.solrs.JavaAsyncSolrClient;
+import org.apache.solr.client.solrj.beans.Field;
+
+class Item {
+    @Field
+    String id;
+    
+    @Field
+    String name;
+
+    @Field("cat")
+    String category;
+
+    @Field
+    float price;
+    
+    TestBean(String id, String name, String category, float price) {
+        this.id = id;
+        this.name = name;
+        this.category = category;
+        this.price = price;
+    }    
+}
+
+JavaAsyncSolrClient solr = JavaAsyncSolrClient.create("http://localhost:8983/solr/collection1");
+Item item1 = new Item("id1", "doc1", "cat1", 10);
+Item item2 = new Item("id2", "doc2", "cat1", 20);
+solr.addBeans(asList(item1, item2))
+    .thenCompose(r -> solr.commit())
+    .thenAccept(r -> System.out.println("beans added"));
+```
+
+[scala]
+```scala
+import io.ino.solrs.AsyncSolrClient
+import io.ino.solrs.future.ScalaFutureFactory.Implicit // or TwitterFutureFactory
+import org.apache.solr.client.solrj.beans.Field;
+
+case class Item(@(Field@field) id: String,
```

Comment:
Needs an `import scala.annotation.meta.field`? Maybe add a space between `Field` and `@field`?